Postgres/Syntax/SchemaDSL.lean

```diff
@@ -0,0 +1,63 @@
+import Postgres.Syntax.InsertDSL
+import Postgres.Syntax.InsertSyntax
+
+namespace SchemaDSL
+
+inductive CreateScope where
+  | default
+  | local
+  | global
+
+inductive NotExistsClause where
```

Comment:
This is isomorphic to `Bool`. Making a new type may provide some additional clarity, but in Lean you can usually also achieve this with named arguments: `foo (ifExists := true)`.